cmd/end2endtest/helpers.go

```diff
@@ -492,71 +509,112 @@ func (t *e2eElection) overwriteVote(choices []int, indexAcct int, waitType strin
 }
 
 // ballotVotes from a default list of 10 vote values that exceed the max value, max total cost and is not unique will
-func ballotVotes(b ballotData, nvotes int) ([][]int, [][]*types.BigInt) {
+func ballotVotes(vop vapi.TallyMode, nvotes int, ballotType string, expectUniqueV bool) ([][]int, [][]*types.BigInt) {
```

Comment:
please move this whole ballotVotes func to ballot.go since it's only used by that test